packages/taco/src/sign.ts

```diff
@@ -173,3 +174,25 @@ export async function signUserOp(
     signingResults: porterSignResult.signingResults,
   };
 }
+
+export async function setSigningCohortConditions(
+  conditions: ConditionExpression,
+  provider: ethers.providers.JsonRpcProvider,
+  domain: Domain,
+  cohortId: number,
+  chainId: number,
+  signer: ethers.Signer,
```

Comment:
A couple things:
1. To remain consistent with `encrypt`, let's take a `Condition` here, internally we can convert it to a `ConditionExpression`.
2. The existing taco APIs (`encrypt`, `decrypt`, `signUserOp`) have an implicit order of parameters that we should try to remain consistent with. `provider` first, `domain` second, ..., `signer` last.

```suggestion
  provider: ethers.providers.JsonRpcProvider,
  domain: Domain,
  conditions: Condition,
  cohortId: number,
  chainId: number,
  signer: ethers.Signer,
```

wdyt?